apps/api/app/Http/Missions/Controllers/MissionBillingController.php

```
@@ -0,0 +1,24 @@
+use App\Domain\Missions\Models\Mission;
+use App\Http\Controllers\Controller;
+use Illuminate\Http\JsonResponse;
+
+class MissionBillingController extends Controller
+{
+    /**
+     * Answers null for a mission billed by the day or the hour: there is no
+     * agreed total to measure progress against, and a zero would read as
+     * "nothing billed" rather than "not applicable".
+     */
+    public function show(Client $client, Mission $mission, SummarizeMissionBilling $summarizeMissionBilling): JsonResponse
```

PHPMD (2.15.0)

[warning] Avoid excessively long variable names like $summarizeMissionBilling. Keep variable name length under 20. (undefined)

(LongVariable)